force-app/main/default/lwc/refundPayment/refundPayment.html

```diff
@@ -17,31 +18,43 @@
         </div>
         <div if:false={isLoading}>
             <div if:false={hasError}>
-                <span>{labels.refundPaymentMessage}</span>
-                <div class="slds-form slds-p-top_medium" role="list">
-                    <div class="slds-form-element slds-form-element_horizontal slds-p-left_none slds-" role="listitem">
-                        <label class="slds-form-element__label slds-text-title_bold slds-p-top_none" for="refund-amount">{labels.refundAmount}</label>
-                        <div class="slds-form-element__control">
-                            <lightning-formatted-number id="refund-amount" value={paymentAmount} format-style="currency" currency-code={currencyCode} currency-display-as="symbol">
-                            </lightning-formatted-number>
-                        </div>
-                    </div>
-                    <div class="slds-form-element slds-form-element_horizontal slds-p-left_none" role="listitem">
-                        <label class="slds-form-element__label slds-text-title_bold slds-p-top_none" for="refund-date">{labels.refundPaymentDate}</label>
-                        <div class="slds-form-element__control">
-                            <lightning-formatted-date-time id="refund-date" value={paymentDate}>
-                            </lightning-formatted-date-time>
-                        </div>
-                    </div>
-                </div>
+                <lightning-layout multiple-rows="true">
+                    <lightning-layout-item size="12">
+                        <label class="slds-text-title_bold slds-p-right_small" for="remaining-balance">
+                            {labels.remainingBalance}:
+                        </label>
+                        <lightning-formatted-number id="remaining-balance"
+                            value={remainingBalance}
+                            format-style="currency"
+                            currency-code={currencyCode}
+                            currency-display-as="symbol">
+                        </lightning-formatted-number>
+                    </lightning-layout-item>
+                    <lightning-layout-item size="4">
+                        <lightning-input class="slds-p-top_large"
+                        data-id="RefundAmountInput"
+                        type="number"
+                        name="refundAmount"
```

Comment:
is there a reason we didn't use the `formatter` property here for `currency`?

Reply:
The main reason is that the `currency` does not accept custom iso-code and will default to the user's local or org-wide currency. This will not work in orgs with multi-currency enabled.